addons/models/auth/AuthService.json

```diff
@@ -0,0 +1,125 @@
+                    "isOptional": false,
+                    "isUnique": true,
+                    "skipScrubbing": true,
+                    "includeInNotification": true,
+                    "indexTypeESFields": {
+                        "text": {
+                            "type": "text",
+                            "analyzer": "atlan_text_analyzer"
+                        }
+                    }
+                },
+                {
+                    "name": "name",
+                    "typeName": "string",
+                    "cardinality": "SINGLE",
+                    "isIndexable": true,
+                    "isOptional": false,
+                    "isUnique": false,
+                    "searchWeight": 10,
+                    "skipScrubbing": true,
+                    "indexTypeESConfig": {
+                        "analyzer": "atlan_text_analyzer"
+                    },
+                    "indexTypeESFields": {
+                        "keyword": {
+                            "type": "keyword",
+                            "normalizer": "atlan_normalizer"
+                        },
+                        "stemmed": {
+                            "type": "text",
+                            "analyzer": "atlan_text_stemmer"
+                        }
+                    }
+                },
+                {
+                    "name": "description",
+                    "typeName": "string",
+                    "cardinality": "SINGLE",
+                    "isIndexable": false,
+                    "isOptional": true,
+                    "isUnique": false,
+                    "searchWeight": 9,
+                    "skipScrubbing": true,
+                    "includeInNotification": true,
+                    "indexTypeESConfig": {
+                        "analyzer": "atlan_text_analyzer"
+                    },
+                    "indexTypeESFields": {
+                        "keyword": {
+                            "type": "keyword",
+                            "normalizer": "atlan_normalizer"
+                        }
+                    }
+                },
+                {
+                    "name": "tenantId",
+                    "typeName": "string",
+                    "cardinality": "SINGLE",
+                    "isIndexable": true,
+                    "isOptional": true,
+                    "isUnique": false,
+                    "skipScrubbing": true
+                },
+                {
+                    "name": "authServiceType",
+                    "typeName": "string",
+                    "indexType": "STRING",
+                    "cardinality": "SINGLE",
+                    "isIndexable": false,
+                    "isOptional": true,
+                    "isUnique": false,
+                    "skipScrubbing": true,
+                    "includeInNotification": true
+                },
+                {
+                    "name": "tagService",
```

Comment:
ENUM

Reply:
Again, Having tagService as an ENUM will not be a good idea since there can be any type of service ideally